hack/grab-profiles.sh

```diff
@@ -65,7 +65,7 @@ output_dir="."
 tunnel_port="${tunnel_port:-1234}"
 
 args=$(getopt -o s:mho:k:c -l server:,master,heapster,output:,kubelet:,scheduler,controller-manager,help,inuse-space,inuse-objects,alloc-space,alloc-objects,cpu,kubelet-binary:,master-binary:,scheduler-binary:,controller-manager-binary:,scheduler-port:,controller-manager-port: -- "$@")
-if [[ $? -ne 0 ]]; then
+if [[ $? ]]; then
```

Comment:
`if ! args=$(getopt -o ...); then` (does this work?)

Reply:
I think that should work too, but this already can work here.